ios/CodePush/CodePush.m

```diff
@@ -760,4 +760,29 @@ - (void)applicationWillResignActive
     resolve(nil);
 }
 
+#pragma mark - Methods for handling dispatching of download progress events to JS (Private)
```

Comment:
If you're fine with ignoring that nitty detail I cna just go ahead and rename that